packages/shared/locales/en.json

```diff
@@ -913,22 +913,29 @@
                 "nft": "NFT"
             }
         },
-        "mintNativeToken": {
```

Comment:
if you change this then every language needs to be re translated, unless you change the key in all language files